terraform/backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
```

Comment:
Why do we need `s3`?

Reply:
We need a backend configuration to manage state files or else the users will either have to host the state locally or b at risk of losing the state of terraform implementation (in simple terms, it means, everytime terraform is initaited, it does not continue from the last deployment, it destroys available resources and create new ones).